crates/subspace-node/src/system_domain/chain_spec.rs

```diff
@@ -71,7 +71,7 @@ pub fn development_config() -> ExecutionChainSpec<GenesisConfig> {
                         min_operator_stake: 100 * SSC,
                     },
                     get_account_id_from_seed("Alice"),
-                    Percent::one(),
+                    Percent::from_percent(10),
```

Comment:
Commit comment says "decrease", but this increases the number, I'm a bit confused here.

Reply:
`Percent::one()` means 100%, `Percent::from_percent(10)` means 10%, so it's decreased in this way.

Reply:
Man, that is a really confusing naming they have here, WTF?!